frontend/packages/ocs-independent-plugin/src/components/dashboard-page/activity-card/card.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import ActivityBody, {
+  RecentEventsBody,
+  OngoingActivityBody,
+} from '@console/shared/src/components/dashboard/activity-card/ActivityBody';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import { FirehoseResult, FirehoseResource } from '@console/internal/components/utils';
+import { EventKind } from '@console/internal/module/k8s';
+import {
+  EventModel,
+  PersistentVolumeClaimModel,
+  PersistentVolumeModel,
+} from '@console/internal/models';
+import { getNamespace } from '@console/shared';
+import { withDashboardResources } from '@console/internal/components/dashboard/with-dashboard-resources';
+import {
+  DATA_RESILIENCY_QUERY,
+  StorageDashboardQuery,
+} from '@console/ceph-storage-plugin/src/constants/queries';
+import { getResiliencyProgress } from '@console/ceph-storage-plugin/src/utils';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+
+const OCS_NS = 'openshift-storage';
```

Comment:
I think this constant can be used on other places too, maybe extract to `src/constants.ts` or something.